tools/global_setup.sh

```diff
@@ -0,0 +1,70 @@
+echo "Starting setup..."
+
+ontology_validator_setup()
+{
+  echo "Starting ontology validator setup"
+  cd validators/ontology_validator
+  sudo python setup.py install
```

Comment:
If sudo privileges are required (I'm not sure they are?), sudo should be removed from these commands and instead the user should be instructed to run the script with sudo.

Reply:
sudo privelages are required to run setup. Will add sudo to the readme